sktime/tests/_config.py

```diff
@@ -60,6 +60,7 @@
     "SARIMAX",
     "StatsModelsARIMA",
     "ShapeletLearningClassifierTslearn",
+    "DartsXGBModel",
```

Comment:
Not just this one, but all these sporadic timeout skips probably affect test coverage quite a lot. Almost all ARIMA versions seem to be here, and I'd be surprised if it's not one of the most used forecaster.

Not blocking, as unrelated to linked issue.